dotnet/samples/Concepts/ChatCompletion/OpenAI_FunctionCalling.cs

```diff
@@ -49,6 +49,24 @@ public async Task AutoInvokeKernelFunctionsMultipleCallsAsync()
         Console.WriteLine(result2[0].Content);
```

Comment:
Is there a special reason for not using the `GetChatMessageContent` extension instead?